### DISCUSSION TITLE
Using OCC in Blender

### DISCUSSION THREAD

**Discussion by CF-8** [IfcOpenShell/IfcOpenShell#4254] [Q&A]
Hi all,
I am really new to OCC and i was wondering if it is possible to use the opencascade libraries within Blender. I am working on Windows and my goal is to run OCC in Blender and to generate the geometry within the Blender viewer. I have created a conda environment from which I am able to run a simple script. In blender i tried importing the OCC libraries by appending their location from within my environment to sys.path but I am always getting an error: ImportError: DLL load failed while importing _Aspect: Module couldnt be found.
I appreciate every help and really hope that it is possible to run OCC in Blender

**Comment by Moult** [comment]
It is possible but you need to do the work to dig out and package the dependencies for OCC yourself. It's been done before, but removed. If you check the commit history of the src/blenderbim/Makefile you should see how it was achieved in the past.
